lib/primitives/object-element.js

```diff
@@ -23,6 +23,29 @@ module.exports = function(BaseElement, ArrayElement, MemberElement) {
       }, {});
     },
 
+    toEmbeddedRefract: function() {
+      if (this.attributes.content.length === 0 && this.meta.content.length === 0) {
+        var refract = {};
+
+        this.forEach(function(value, key) {
+          refract[key.toValue()] = value.toEmbeddedRefract();
+        });
+
+        return refract;
+      }
+
+      return {
+        refract: this.toRefract()
+      };
+    },
+
+    fromEmbeddedRefract: function(embeddedObj) {
```

Comment:
Can you explain? The `fromEmbeddedRefract` is inherited, but this overwrites it since the Object Element acts a little different. On normal elements, the properties are stored as attributes, but for Object Elements, they are not since it is an actual object.